package.json

```diff
@@ -113,32 +113,35 @@
         "@rollup/plugin-node-resolve": "^15.2.3",
         "@sindresorhus/slugify": "^2.1.1",
         "@spectrum-web-components/eslint-plugin": "file:./linters/eslint",
-        "@storybook/addon-a11y": "^7.5.0",
-        "@storybook/addon-designs": "^8.0.0",
-        "@storybook/addon-essentials": "^7.5.0",
-        "@storybook/addon-links": "^7.5.0",
-        "@storybook/web-components": "^7.5.0",
```

Comment:
looks like this package was removed but this is how we set our custom manifest (the part you commented out in preview.js). Was this intentional or is it not supported in storybook 8?

Reply:
Does look like it comes with v8 support: https://www.npmjs.com/package/@storybook/web-components

Reply:
It may say that it's supported but there's some serious issues in that package. When I add the dependency, it adds this to our install logs (yes, it does the same if we bump all packages to 8.6.12 too):

<img width="2221" alt="Screenshot 2025-04-14 at 1 38 53 PM" src="https://github.com/user-attachments/assets/f7c48e01-db16-407a-9462-5fd44e4a2d50" />

When we add it in anyway despite the logged error, it blocks styles from loading. When I commented out the CEM loading, the arg types still loaded in the stories and the styles rendered. We can debug loading the CEM another way but I'm not sure this package is going to be it.